divviup/build.gradle.kts

```diff
@@ -72,3 +72,14 @@ tasks.withType<Test>().matching { it.name.matches(Regex("test.*UnitTest"))}.conf
     val capitalizedHostRustTarget = hostRustTarget.replaceFirstChar { it.uppercase() }
     dependsOn(tasks.named("cargoBuild${capitalizedHostRustTarget}"))
 }
+
+afterEvaluate {
```

Comment:
Q: why does this need to be in an `afterEvaluate` block?

Reply:
This is necessary for `android.libraryVariants`. If we were to run this earlier, that set would be empty, but waiting for after evaluation lets AGP set up the debug and release variants first.